src/lib/marlin_plonk_bindings/stubs/src/tweedle_dum.rs

```diff
@@ -64,6 +64,18 @@ pub fn caml_tweedle_dum_rng(i: ocaml::Int) -> CamlTweedleDum {
     CamlTweedleDum(UniformRand::rand(&mut rng))
 }
 
+#[ocaml::func]
+pub extern "C" fn caml_tweedle_dum_endo_base() -> CamlTweedleFp {
+    let (endo_q, _endo_r) = commitment_dlog::srs::endos::<GAffine>();
+    CamlTweedleFp(endo_q)
+}
+
+#[no_mangle]
```

Comment:
why does the annotation here differ from the one above?

Reply:
Woops! Fixing.